src/bufr/BufrParser/Exports/Variables/SpectralRadianceVariable.cpp

```diff
@@ -0,0 +1,178 @@
+/*
+ * (C) Copyright 2020 NOAA/NWS/NCEP/EMC
```

Comment:
copyright date should be 2022

Reply:
Done!  Changed to 2022.

Reply:
See Fortran code section in the description.